### DISCUSSION TITLE
Pop-up with dynamic content

### DISCUSSION THREAD

**Discussion by OiseauParadigme** [elementor/elementor#29007] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Create pop-up with dynamic content from custom post type and advanced custom fields (ACF)

### Describe the solution you'd like.

Hello,

For a blog, I created custom post types associated with advanced custom files, which I display on the home page of my client's website. 
One of these CPTs contains YouTube videos; I need that when clicking on the publication, a pop-up opens to display the video in large size in order to avoid making links to individual posts. Navigation on the website would be greatly improved.

I tried everything but without any results, and I read on other threads (https://github.com/elementor/elementor/issues/7106) that Elementor Pro does not provide this functionality since 5 years (closed thead).

So is this feature really not included with Elementor Pro ? When could we expect to see it arrive ?
Are there alternatives without having to subscribe to other paid features like Jet, which would not require any code ?

Thank you in advance for your feedback.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
Duplicate, already submitted at https://github.com/orgs/elementor/discussions/25473

**Comment by Mesmer99** [comment]
So the closest way I've get there is using off-canvas widget as I mentioned in https://github.com/orgs/elementor/discussions/28935
It opens multiple off canvas but dynamic funtioanality works so I think guys are on the way of adding that.

**Reply by OiseauParadigme** [reply]
Thank you very much, I will try this !

**Reply by Mesmer99** [reply]
They released this feature in 3.26 beta
